localization/languages/ca.json

```diff
@@ -0,0 +1,299 @@
+    "appMenuServices": "Serveis",
+    "appMenuHide": "Amagar %n",
+    "appMenuHideOthers": "Amagar altres",
+    "appMenuShowAll": "Mostrar tot",
+    "appMenuQuit": "Sortir de %n",
+    "appMenuBringToFront": "Portar tot al davant",
+    /* PDF Viewer */
+    "PDFPageCounter": {
+      "unsafeHTML": "pàgina <input type='text'/> de <span id='total'></span>"
+    },
+    /* Context Reader */
+    "buttonReaderSettings": "Configuració del lector",
+    "buttonReaderLightTheme": "Clar",
+    "buttonReaderSepiaTheme": "Sepia",
+    "buttonReaderDarkTheme": "Fosc",
+    "openReaderView": "Obrir sempre en vista de lector",
+    "autoRedirectBannerReader": "Obrir sempre articles d'aquest lloc en vista de lector?",
+    "buttonReaderRedirectYes": "Sí",
+    "buttonReaderRedirectNo": "No",
+    "articleReaderView": "Article original",
+    /* Download manager */
+    "downloadCancel": "Cancel·lar",
+    "downloadStateCompleted": "Completat",
+    "downloadStateFailed": "Fallat",
+    /* Update Notifications */
+    "updateNotificationTitle": "Hi ha una nova versió de Min disponible",
+    /* Autofill settings */
+    "settingsPasswordAutoFillHeadline": "Autocompletat de contrasenyes",
+    "settingsSelectPasswordManager": "Tria un dels gestors de contrasenyes suportats:",
+    "keychainViewPasswords": "Veure contrasenyes desades",
+    /* Password manager setup */
+    "passwordManagerSetupHeading": "Acaba de configurar %p per utilitzar l'autocompletat",
+    "passwordManagerSetupStep1": {
+      "unsafeHTML": "Primer, <a id='password-manager-setup-link'></a> i extreu-ho pel teu sistema."
+    },
+    "passwordManagerInstallerSetup": {
+      "unsafeHTML": "Descarrega <a id='password-manager-setup-link-installer'></a> i arrossega el fitxer a la caixa de sota:"
+    },
+    "passwordManagerSetupLink": "descarrega l'eina CLI de %p",
+    "passwordManagerSetupLinkInstaller": "l'instal·lador CLI de %p",
+    "passwordManagerSetupStep2": "Després, arrossega l'eina a la caixa de sota:",
+    "passwordManagerSetupDragBox": "Arrossega l'eina aquí",
+    "passwordManagerSetupInstalling": "Instal·lant...",
+    "passwordManagerBitwardenSignIn": "Per connectar el teu compte Bitwarden, ves a vault.bitwarden.com/#/settings/account, desplaça't fins al final, i tria \"Veure clau API\". Després, enganxa els valors als camps de sota.",
+    "passwordManagerSetupSignIn": "Inicia sessió al teu gestor de contrasenyes per començar a utilitzar l'autocompletat. Les teves credencials no es desaran enlloc dins de Min.",
+    "disableAutofill": "Deshabilitar autocompletat",
+    "passwordManagerSetupUnlockError": "No s'ha pogut desbloquejar el magatzem de contrasenyes: ",
+    "passwordManagerSetupRetry": "Assegura't d'utilitzar el fitxer correcte i d'introduir una contrasenya correcta. Pots tornar a intentar-ho arrossegant l'eina aquí de nou.",
+    "passwordManagerUnlock": "Introdueix la teva contrasenya per utilitzar l'autocompletat:",
```

Comment:
This has a parameter for the manager name in the original:

https://github.com/minbrowser/min/blob/6ab87e48f82aebf215e268910a4635130ffd4243/localization/languages/en-US.json#L279